upstream-sha256: 789e2a42356034e426914b6a4736749962c1115b7556e2efeb8ec34ba9aa0819
actual-sha256: 916e888ce49bb8ced3eda3e94dbca559ad0aa40de9dfa96a2d4455279a1aa2b1
--- upstream.expected
+++ tolk.actual
@@ -88,8 +88,8 @@
   float4 wmma1 = __WMMA_16_16_32___bf16_float(make_hip_bfloat168(val18,val19,val20,val21,val22,val23,val25,val26), make_hip_bfloat168(val54,val55,val56,val57,val58,val59,val61,val62), cast0, 0, 0, 0);
   float4 wmma2 = __WMMA_16_16_32___bf16_float(make_hip_bfloat168(val27,val28,val29,val30,val1,val2,val3,val4), make_hip_bfloat168(val32,val33,val34,val35,val37,val38,val39,val40), cast0, 0, 0, 0);
   float4 wmma3 = __WMMA_16_16_32___bf16_float(make_hip_bfloat168(val31,val0,val5,val6,val15,val16,val17,val24), make_hip_bfloat168(val63,val36,val41,val42,val51,val52,val53,val60), cast0, 0, 0, 0);
-  *(data2_16384+(alu6+128)) = (wmma3.y+wmma0.y+wmma1.y+wmma2.y);
-  *(data2_16384+(alu6+256)) = (wmma3.z+wmma0.z+wmma1.z+wmma2.z);
-  *(data2_16384+(alu6+384)) = (wmma3.w+wmma0.w+wmma1.w+wmma2.w);
-  *(data2_16384+alu6) = (wmma3.x+wmma0.x+wmma1.x+wmma2.x);
+  *(data2_16384+(alu6+128)) = (0.0f+wmma3.y+wmma0.y+wmma1.y+wmma2.y);
+  *(data2_16384+(alu6+256)) = (0.0f+wmma3.z+wmma0.z+wmma1.z+wmma2.z);
+  *(data2_16384+(alu6+384)) = (0.0f+wmma3.w+wmma0.w+wmma1.w+wmma2.w);
+  *(data2_16384+alu6) = (0.0f+wmma3.x+wmma0.x+wmma1.x+wmma2.x);
 }
